cvat/schema.yml

```diff
@@ -7385,6 +8171,170 @@ components:
         * `AZURE_CONTAINER` - AZURE_CONTAINER
         * `GOOGLE_DRIVE` - GOOGLE_DRIVE
         * `GOOGLE_CLOUD_STORAGE` - GOOGLE_CLOUD_STORAGE
+    QualityReport:
+      type: object
+      properties:
+        id:
+          type: integer
+          readOnly: true
+        job_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        task_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        parent_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        target:
+          $ref: '#/components/schemas/QualityReportTarget'
+        summary:
+          $ref: '#/components/schemas/QualityReportSummary'
+        parameters:
+          $ref: '#/components/schemas/QualityReportParameters'
+        created_date:
+          type: string
+          format: date-time
+          readOnly: true
+        target_last_updated:
+          type: string
+          format: date-time
+          readOnly: true
+        gt_last_updated:
+          type: string
+          format: date-time
+          readOnly: true
+      required:
+      - parameters
+      - summary
+      - target
+    QualityReportCreateRequest:
+      type: object
+      properties:
+        task_id:
+          type: integer
+          writeOnly: true
+      required:
+      - task_id
+    QualityReportParameters:
```

Comment:
It seems odd to have both parameters _and_ settings. What is the relation between the two?

Reply:
> not every parameter can be controlled

If something can't be controlled, how is it a parameter? It seems to me that being controllable is inherent to the definition of a parameter.

> and not everything controlled is supposed to be related to reports or quality estimation process.

I don't understand this either; we're talking about `QualitySettings`, shouldn't everything in there affect the quality estimation process?

Reply:
Quality control settings can include such parameters as report computation period or default output format, which do not affect any quality metrics. I suppose settings may be split into more specific sections if such parameters are introduced. I think, we can remove the report parameters from the response, as we did not find any good use for them in UI yet.

Reply:
> I think, we can remove the report computation parameters from the response, as we did not find any good use for them in UI yet.

That's fine by me.

But FWIW, I think we shouldn't intermix settings that affect the contents of the report with other settings. Ideally, the former should be aggregated into a dedicated structure that can then be reused in the quality report.

Reply:
Removed parameters from reports summary